src/FunctionApps/python/IntakePipeline/requirements.txt

```diff
@@ -0,0 +1,3 @@
+# Do not include azure-functions-worker as it may conflict with the Azure Functions platform
+
+azure-functions
```

Comment:
we've got reqiurements.txt in the python function app root

Reply:
Removed this instance of it.